### ISSUE TITLE
plugin: accessibility linter

### ISSUE THREAD

**Issue by DirtyF** [11ty/eleventy#418]
`11ty-plugin-a11y` can lint output. It displays warnings about accessibility (missing alt, headings, etc.)

/cc @chrisdmacrae

**Comment by Ryuno-Ki** [comment]
Do you have a preferred accessibility linter which this one would call?

**Comment by bjankord** [comment]
[axe-core](https://github.com/dequelabs/axe-core) is a popular tool that offers reporting and is well maintained. [Google picked it for use in their dev tools back in 2017](https://www.deque.com/blog/google-selects-deques-axe-chrome-devtools/).

**Comment by Ryuno-Ki** [comment]
I know :-)
What about [tenon](https://tenon.io/) or [tota11y](https://khan.github.io/tota11y/)?

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
I did an accessibility tooling review recently and landed on `axe` for a major project. I’d prefer that one if possible here.
